src/plots/plots.js

```diff
@@ -540,6 +544,23 @@ plots.supplyDefaults = function(gd) {
             (gd.calcdata[i][0] || {}).trace = trace;
         }
     }
+
+    // Set up the default keyframe if it doesn't exist:
+    if(!gd._frameData) {
+        gd._frameData = {};
+    }
+
+    if(!gd._frameData._frames) {
+        gd._frameData._frames = [];
+    }
+
+    if(!gd._frameData._frameHash) {
+        gd._frameData._frameHash = {};
+    }
+
+    if(!gd._frameData._counter) {
+        gd._frameData._counter = 0;
+    }
```

Comment:
We currently have this [`graphWasEmpty`](https://github.com/plotly/plotly.js/blob/0725ea40027ae0a4be80805d0a413cd732d3bf35/src/plot_api/plot_api.js#L91) check.